public/specifications/monitoring/MONITORING.md

```diff
@@ -18,6 +18,19 @@ Pillarbox integrates with a monitoring platform that provides real-time and hist
 
 This article describes the flexible event model that allows our players to send data to our monitoring platform.
 
+## JSON Key / Value Naming Conventions
+
+The following naming conventions are applied for key / values appearing in JSON payloads:
+
+- Keys use snake case.
+- Values are provided in a form best suited for display, usually capitalized, with possible exceptions when brand
+  names are involved (e.g. macOS must not be capitalized as MacOS).
+- The only values provided in uppercase are event types (e.g. `START`).
+
+> [!IMPORTANT]
```

Comment:
Docsify doesn't seem to like `IMPORTANT`? 🤔 
<img width="707" alt="Screenshot 2024-09-24 at 14 11 48" src="https://github.com/user-attachments/assets/7232f7c9-6089-415e-ab03-a94729504476">

https://srgssr.github.io/pillarbox-documentation/pr-preview/pr-99/#/specifications/monitoring/MONITORING?id=json-key-value-naming-conventions

Reply:
[My mistake](https://github.com/fzankl/docsify-plugin-flexible-alerts), will fix, thanks.

Reply:
https://github.com/fzankl/docsify-plugin-flexible-alerts/issues/36
One day 🤞🏻